setup.sh

```diff
@@ -264,14 +264,12 @@ kubectl --namespace dynatrace \
     --from-literal=clientId=$DYNATRACE_OAUTH_CLIENT_ID \
     --from-literal=clientSecret=$DYNATRACE_OAUTH_CLIENT_SECRET
 
-kubectl apply -f ./observability/dynatrace/crossplane-dashboard
+helm repo add crossplane-observability-demo-dashboards https://katharinasick.github.io/crossplane-observability-demo-dashboards
 
-set +e
```

Comment:
I moved this delete to the destroy script a while back (just like you did with the AWS secrets)
I hope that's ok? Otherwise, I don't mind adding it here again :)